docs/source/reference/hi.rst

```diff
@@ -0,0 +1,29 @@
+.. _hi:
+
+IMAP-Hi
+==========================
+
+This is the IMAP-Hi Instrument module, which contains the code for processing
+data from the IMAP-Hi instrument.
+
+The L0 code to decommutate the CCSDS packet data can be found below.
+
+.. currentmodule:: imap_processing
```

Comment:
This should be documented in the root level `imap_processing` location, not within `hi`.